spec/javascripts/stripPII_spec.js

```diff
@@ -1,50 +1,53 @@
+/* eslint-env jasmine */
+/* global stripPII */
+
 describe('stripPII', function () {
-  it ('returns an empty string given an empty string', function () {
+  it('returns an empty string given an empty string', function () {
     expect(stripPII('')).toEqual('')
   })
 
-  it ('returns an empty string given a null', function () {
+  it('returns an empty string given a null', function () {
     expect(stripPII(null)).toEqual('')
   })
 
-  it ('replaces nothing if there is no query string', function () {
+  it("does nothing if there's no query string", function () {
     var givenURL = new URL('http://example.com/')
     var expectedURL = 'http://example.com/'
 
     expect(stripPII(givenURL)).toEqual(expectedURL)
   })
 
-  it ('replaces nothing if there are no redactable params', function () {
+  it('redacts a single parameter', function () {
     var givenURL = new URL('http://example.com/?test=blah')
-    var expectedURL = 'http://example.com/?test=blah'
+    var expectedURL = 'http://example.com/?test=<TEST>'
 
     expect(stripPII(givenURL)).toEqual(expectedURL)
   })
 
-  it ('replaces reference_number', function() {
+  it('redacts reference_number', function () {
```

Comment:
I don't think this test is needed any more as the overall functionality is covered by the `redacts all parameters` test below.

Reply:
Good point! I've now removed it.